java/mtsj/core/pom.xml

```diff
@@ -136,6 +136,11 @@
       <artifactId>${mts.database.artifactId}</artifactId>
       <version>${mts.database.version}</version>
     </dependency>
+    <dependency><!-- Need for SAP Hana-->
+      <groupId>com.sap.cloud.db.jdbc</groupId>
+      <artifactId>ngdbc</artifactId>
+      <version>2.3.48</version>
```

Comment:
just as a navie question: cant we just move this to the hana db profile? I assume the code has some direct dependencies on this. Should we then (later) extract the feature to its own maven module?

Reply:
Will be done after merge